metric_config_parser/sql.py

```diff
@@ -110,7 +129,14 @@ def generate_data_source_sql(
         .render(
             **{
                 "data_source": data_source_definition,
+                "data_sources": {
+                    slug: data_source
+                    for definition in config_collection.definitions
+                    for slug, data_source in definition.spec.data_sources.definitions.items()
+                    if platform == definition.platform
+                },
                 "where": where,
+                "select_fields": select_fields,
```

Comment:
Should `select_fields` appear in the template somewhere? I'm not seeing where this is used.

Reply:
It somehow got lost in the SQL. Added it back in